go/internal/configschema/mutation_test.go

```
@@ -0,0 +1,218 @@
+		t.Fatalf("generation = %d, %v", got, err)
+	}
+}
+
+func TestMutationCoordinatorFailsImmediatelyWhileAnotherWriterHoldsImmediate(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "config.json")
+	db, err := sql.Open("sqlite", MutationDatabasePath(path))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+	if _, err := db.Exec("BEGIN IMMEDIATE"); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)